src/server/game/Chat/Channels/Channel.h

```diff
@@ -224,6 +224,9 @@ class Channel
     void AddWatching(Player* p);
     void RemoveWatching(Player* p);
 
+    // acore
+    void ToggleModeration(Player* p); 
```

Comment:
why the `acore` comment?

Reply:
Thought we'd split the functions in groups for reference purposes, e.g

// trinity

// sunwell

// acore

but it's pointless, so I removed it